lib/Serializer.js

```diff
@@ -0,0 +1,118 @@
+class Serializer {
+	constructor() {
+		this._serializers = {_default: _defaultSerializer};
+		this._defaultSerializer = "_default";
```

Comment:
I really want to move to private properties. Happen to know what version of Node.js supports that? This just feels kinda messy to me. Lots of other cases in the code base where we could do that as well.

Reply:
12.16.1 is the latest with private properties. Which is the current LTS and used by Lambda.

Reply:
@ablankenship10 Happen to know if AWS Lambda allows for creating a new Lambda function with a Node.js version below that?

Reply:
Lambda currently supports version 10 and 12 only.

Reply:
Fixed in 3dd621a04d5ec3c93e97a1ea6f90a538603c5801